[VarDumper] Add support of named arguments to `dd()` and `dump()` to display a label (#48432)
symfony/symfony: src/Symfony/Component/VarDumper/CHANGELOG.md

CHANGELOG
=========

6.3
---

>  * Add support of named arguments to `dd()` and `dump()` to display a "section name"

Reviewer: a "label"?

Reviewer 2: The function can be named according to what it actually does.
```suggestion
 * Add support of named arguments to `dd()` and `dump()` to display the argument name.
```
